arangod/Cluster/ClusterInfo.cpp

```diff
@@ -570,13 +582,22 @@ void ClusterInfo::loadPlan() {
           if (vocbase == nullptr) {
             // No database with this name found.
             // We have an invalid state here.
+            LOG_TOPIC(ERR, Logger::AGENCY)
+              << "No database '" << databaseName << "' found,"
+                 " corresponding view will be ignored for now and the invalid information"
+                 " will be repaired. VelocyPack: " << viewsSlice.toJson();
             continue;
           }
 
           for (auto const& viewPairSlice :
                VPackObjectIterator(viewsSlice)) {
             VPackSlice const& viewSlice = viewPairSlice.value;
             if (!viewSlice.isObject()) {
+              LOG_TOPIC(ERR, Logger::AGENCY)
```

Comment:
and here

Reply:
I would vote for INFO, again if "it will be repaired" automatically.